apps/nextjs-website/src/app/[productSlug]/overview/page.tsx

```diff
@@ -121,6 +121,7 @@ const OverviewPage = async ({ params }: ProductParams) => {
         useHoverlay={false}
         altText={hero.altText}
         theme='light'
+        gridTextSx={{ marginTop: { xs: '62px', md: '77px' } }}
```

Comment:
I think this `gridTextSx` should also set for api, quickStart, guides and tutorials pages?

Reply:
My bad i see Hero is only used in overview page